package.json

```diff
@@ -3,87 +3,59 @@
     "@emotion/css": "^11.0.0",
-    "@emotion/react": "^11.1.1",
+    "@emotion/react": "^11.1.2",
     "@emotion/styled": "^11.0.0",
-    "@fortawesome/fontawesome-svg-core": "^1.2.32",
-    "@fortawesome/free-brands-svg-icons": "^5.15.1",
-    "@fortawesome/free-regular-svg-icons": "^5.15.1",
-    "@fortawesome/free-solid-svg-icons": "^5.15.1",
-    "@fortawesome/react-fontawesome": "^0.1.13",
-    "@testing-library/jest-dom": "^5.11.6",
-    "@testing-library/react": "^11.2.2",
-    "@testing-library/user-event": "^12.4.0",
-    "@types/jest": "^26.0.16",
-    "@types/node": "^12.19.8",
-    "@types/react-dom": "^16.9.10",
     "axios": "^0.21.0",
     "axios-case-converter": "^0.6.0",
-    "core-js": "^3.8.1",
-    "emotion": "^11.0.0",
     "immer": "^8.0.0",
-    "storybook": "^6.1.10",
-    "typescript": "^4.1.2",
     "webfontloader": "^1.6.28"
   },
   "devDependencies": {
-    "@babel/cli": "^7.12.8",
     "@babel/core": "^7.12.9",
     "@babel/plugin-transform-runtime": "^7.12.1",
     "@babel/preset-env": "^7.12.7",
     "@babel/preset-react": "^7.12.7",
     "@babel/preset-typescript": "^7.12.7",
+    "@fortawesome/fontawesome-svg-core": "^1.2.32",
+    "@fortawesome/free-brands-svg-icons": "^5.15.1",
+    "@fortawesome/free-regular-svg-icons": "^5.15.1",
+    "@fortawesome/free-solid-svg-icons": "^5.15.1",
+    "@fortawesome/react-fontawesome": "^0.1.13",
     "@rollup/plugin-babel": "^5.2.2",
     "@rollup/plugin-node-resolve": "^11.0.0",
-    "@rollup/plugin-typescript": "^8.0.0",
-    "@storybook/addon-actions": "^6.1.10",
-    "@storybook/addon-essentials": "^6.1.10",
-    "@storybook/addon-links": "^6.1.10",
-    "@storybook/node-logger": "^6.1.10",
     "@storybook/react": "^6.1.10",
-    "@types/react": "^16.14.2",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
     "@types/webfontloader": "^1.6.32",
-    "@typescript-eslint/eslint-plugin": "^4.9.0",
-    "@typescript-eslint/parser": "^4.9.0",
-    "babel-plugin-optimize-clsx": "^2.6.1",
-    "eslint": "^7.11.0",
-    "eslint-config-prettier": "^6.15.0",
-    "eslint-plugin-flowtype": "^5.2.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jsx-a11y": "^6.3.1",
+    "@typescript-eslint/eslint-plugin": "^4.9.1",
+    "@typescript-eslint/parser": "^4.9.1",
+    "eslint": "^7.15.0",
+    "eslint-config-prettier": "^7.0.0",
     "eslint-plugin-prettier": "^3.2.0",
     "eslint-plugin-react": "^7.21.5",
     "eslint-plugin-react-hooks": "^4.2.0",
     "prettier": "^2.2.1",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "rollup": "^2.34.2"
+    "react": "17.0.1",
+    "react-dom": "17.0.1",
+    "rollup": "^2.34.2",
+    "typescript": "~4.1.2"
   },
   "peerDependencies": {
     "react": "^16.14 || ^17.0",
     "react-dom": "^16.14 || ^17.0"
   },
   "scripts": {
-    "build": "rollup --config & tsc",
-    "storybook": "start-storybook -p 6006",
+    "build": "rollup --config && tsc",
+    "start": "start-storybook -p 6006",
```

Comment:
I use `npm start` in a lot of projects, it's quicker to type and is a common command in projects.